src/query-builder/transformer/RawSqlResultsToEntityTransformer.ts

```diff
@@ -146,6 +147,36 @@ export class RawSqlResultsToEntityTransformer {
         return hasData;
     }
 
+    protected transformComputedColumns(rawResults: any[], alias: Alias, entity: ObjectLiteral): boolean {
+        let hasData = false;
+
+        this.expressionMap.computedSelects.forEach(computedSelect => {
+            if (computedSelect.aliasName && computedSelect.mapToProperty) {
+                const fields: any[] = computedSelect.mapToProperty.split("_");
```

Comment:
What if the property has underscores in the name?

Reply:
This would be a problem, because it would consider it as an object within another. We can define an option to personalize the separator of the chain and by default is `_`.